mobile/src/bundles/common/components/image-picker/image-picker.tsx

```diff
@@ -54,16 +54,16 @@ const ImagePicker: React.FC<ImagePickerProperties> = ({
             }
             if (grantedCamera === PermissionsAndroid.RESULTS.NEVER_ASK_AGAIN) {
                 notifications.showError({
-                    title: 'Camera permission denied.',
-                    text: 'Please provide permission to access your camera in settings',
+                    text1: 'Camera permission denied.',
```

Comment:
text1, text2 ?

I think it is not good naming

Reply:
![sds](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/112649713/5e0285bf-c40e-4fbc-ae9a-bdaf09d3629d)
Because this is in the documentation [react-native-toast-message](https://github.com/calintamas/react-native-toast-message/tree/main).

Reply:
It seems to me that this is because the ShowMessageArguments type is not specified.
Maybe this will work?
```
const notifications.showError({
    title: 'Camera permission denied,
    text: 'Please provide permission to access your camera in settings',
} as ShowMessageArguments);
```

Reply:
we can refactor framework/notifications :
 `public showError(settings: ShowMessageArguments): void {
        this.notify.show({
            type: 'error',
            ...settings,
        });
    }`
    to: 
    `    public showError(settings: ShowMessageArguments): void {
        const {title, text} = settings
        this.notify.show({
            type: 'error',
           text1: title,
           text2: text
        });
    }
    `
    the same to showSuccess method.
    And return everything back with ShowMessageArguments type and values in the components. I have make mistake with destructurisation in this component...

Reply:
![aaa](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/112649713/0821f573-f778-49cf-9975-a36e562db218)
where ...setting should be text1, text2 and we receive title, text. It could wok if we change: 
![sdsd](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/112649713/4f2fa58a-7ae2-443e-b046-342c3a041b22)

Reply:
So, maybe it is better?
Leave **title** and **text** everywhere, and add what you suggested here
```
    public showError(settings: ShowMessageArguments): void {
        this.notify.show({
            type: 'error',
            text1: settings.title,
            text2: settings.text,
        });
    }
```